ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -870,4 +802,132 @@ long getAllocatedUsableCapacity() {
       return clusterWideAllocatedUsableCapacityBytes.get();
     }
   }
+
+  private class DatacenterInitializer implements Runnable {
```

Comment:
Moved it out into its own class. Unfortunately, I had to pass a lot of variables in to make it work, but this can probably be refactored a bit further in the future when SimpleClusterChangeHandler is removed